libs/dh/admin/feature-permissions/src/lib/details/dh-admin-permission-detail.component.html

```diff
@@ -43,28 +44,24 @@ <h2>{{ permission?.name }}</h2>
               <watt-description-list-item [label]="t('permissionName')" [value]="permission.name" />
               <watt-description-list-item
                 [label]="t('permissionDescription')"
-                [value]="permission.description"
+                [value]="selectedPermission.description"
               />
             </watt-description-list>
           </watt-card>
         </watt-tab>
         <watt-tab [label]="t('tabs.userRoles.tabLabel')">
-          <dh-admin-permission-roles [selectedPermission]="permission" />
+          <dh-admin-permission-roles [selectedPermission]="selectedPermission" />
         </watt-tab>
         <watt-tab [label]="t('tabs.marketRoles.tabLabel')">
-          <dh-admin-permission-market-roles [selectedPermission]="permission" />
+          <dh-admin-permission-market-roles [selectedPermission]="selectedPermission" />
         </watt-tab>
         <watt-tab
           *dhPermissionRequired="['user-roles:manage']"
           [label]="t('tabs.history.tabLabel')"
         >
-          <dh-admin-permission-audit-logs [selectedPermission]="permission" />
+          <dh-admin-permission-audit-logs [selectedPermission]="selectedPermission" />
         </watt-tab>
       </watt-tabs>
     </watt-drawer-content>
   }
 </watt-drawer>
-
-@if (isEditPermissionModalVisible && permission) {
-  <dh-edit-permission-modal [permission]="permission" (closed)="modalOnClose($event)" />
-}
```

Comment:
praise: love it when we can instantiate the component dynamically instead of manually keeping track of shown/not shown state.